r/R/dataset-write.R

```diff
@@ -54,6 +54,40 @@
 #' - `null_fallback`: character to be used in place of missing values (`NA` or
 #' `NULL`) when using Hive-style partitioning. See [hive_partition()].
 #' @return The input `dataset`, invisibly
+#' @examples
+#' # We can partition by one more variables, say, cyl and gear in mtcars dataset
+#' one_part_dir <- tempfile()
+#' two_part_dir <- tempfile()
+#' write_dataset(mtcars, one_part_dir, "parquet", partitioning = "cyl")
+#' write_dataset(mtcars, two_part_dir, "parquet", partitioning = c("cyl", "gear"))
```

Comment:
```suggestion
#' write_dataset(mtcars, one_part_dir, partitioning = "cyl")
#' write_dataset(mtcars, two_part_dir, partitioning = c("cyl", "gear"))
```

We don't need to (and shouldn't) specify the format argument here, we changed that so the default is parquet, so we don't need to specify it here.